Library/Config.ts

```diff
@@ -109,6 +124,62 @@ class Config {
         }
     }
 
+    private _generateConfigurationObjectFromEnvVars() {
```

Comment:
Is this actually using env vars somewhere?, name is misleading

Reply:
Yeah.. the `_config` object is reading configs from environment variables. Should I update it to `_generateConfigurationObject`?